che-editors.yaml

```diff
@@ -87,8 +87,8 @@ editors:
           - exposedPort: 13132
           - exposedPort: 13133
         memoryLimit: "512M"
-        cpuLimit: 1500m
-        cpuRequest: 500m
+        cpuLimit: 1000m
```

Comment:
What's the reason to lower the CPU Limit?

Reply:
That will decrease Theia performances without any real benefit (on minikube it works fine with cpu limit set to 1500).

Reply:
The reason why CPU Limit was decreased was monitoring overall consumption of CPU by Theia container, which was around 0.2 CPU for all workspace lifecycle. Created revert commit, that sets up previous value for the cpuLimit: https://github.com/eclipse/che-plugin-registry/pull/833. So the only cpuRequest value is being updated.

Reply:
Thank you